VariationalBayes/test_exponential_families.py

```diff
@@ -157,17 +157,14 @@ def test_logitnormal_moments(self):
         np_test.assert_allclose(
             prior_samples,
             ef.get_e_dp_prior_logitnorm_approx(
-                        alpha, lognorm_means, lognorm_infos, GH_LOC, GH_WEIGHTS),
+                        alpha, lognorm_means, lognorm_infos, gh_loc, gh_weights),
             atol = tol3)
 
-        # Was this what you meant when you wanted to check the autodiff?
-        x = np.random.normal(0, 1e6, size = 10)
+        x = np.random.normal(0, 1e2, size = 10)
```

Comment:
So it turns out even with my funky way of computing log(expit(x)), the gradient will still be NaN when x is smaller than -1e2, since -np.log(1 + np.exp(-x)) will be -Inf. 

But this will be an issue even if I just wrote np.log(sp.special.expit(x)). 

I think then the advantage of the way I wrote it is, even though we can't evaluate the gradient past x < -1e2, the function itself actually still returns a number for such small x. Though idk if that even matters.